### PR TITLE
Adding proxy and timeout options to AbstractScraper.

### PR DESCRIPTION
**Pull request by afichtenholtz**
Allows us to specify proxy server and timout options when calling scrape_me().

### CODE REVIEW

**Comment by afichtenholtz** [comment]
Looks like the build is failing due to a formatting issue...is this due to my inadvertent use of tabs instead of spaces in my initial commit?

**Comment by bfcarpio** [comment]
If you fixed those spacing issues in the second commit then it's probably something else. Did you setup your dev environment as described in the README? We have a nice tool that'll run the same style / quality checks locally before every commit. You can also just run the tool `black` manually on the files.

**Comment by afichtenholtz** [comment]
You were absolutely correct about the line length. Checks now pass successfully.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ repos:
     rev: 19.3b0
     hooks:
       - id: black
--   repo: https://gitlab.com/pycqa/flake8
+-   repo: https://gitlab.com/pycqa/flake8.git
```
Comment (bfcarpio) [inline .pre-commit-config.yaml:12]: Was this intended? Is this needed?

**Review by bfcarpio — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (afichtenholtz) [reply .pre-commit-config.yaml:12]: This is required to pull flake8 on CentOS 7 as documented in https://github.com/pre-commit/pre-commit/issues/1206

Comment (bfcarpio) [reply .pre-commit-config.yaml:12]: Nope, just wondering why that file was changed. Getting that bug linked is good documentation for us if something goes awry.

Thanks for your contribution to the package!

**Review by bfcarpio — APPROVED** [review]

**Comment by bfcarpio** [comment]
@hhursev I think this is a clean solution, but since it's not just adding a scraper I'll leave this for your approval.

**Comment by hhursev** [comment]
It's awesome :) Thanks 🍻  Merging and bumping version

**Pull request merged**